nextflow.config

```diff
@@ -19,48 +19,65 @@ params {
     igenomes_ignore            = true
 
     // Annotation options
-    annotation_tool                 = 'prodigal'
-    save_annotations                = false
+    annotation_tool                     = 'prodigal'
+    save_annotations                    = false
     
-    annotation_prodigal_singlemode  = false
-    annotation_prodigal_closed      = false
-    annotation_prodigal_transtable  = 11
-    annotation_prodigal_forcenonsd  = false
-
-    annotation_bakta_db             = null
-
-    annotation_prokka_metagenome    = true
-    annotation_prokka_rawproduct    = false
-    annotation_prokka_kingdom       = 'Bacteria'
-    annotation_prokka_gcode         = 11
-    annotation_prokka_cdsrnaolap    = false
-    annotation_prokka_rnammer       = false
-    annotation_prokka_mincontiglen  = 1
-    annotation_prokka_evalue        = 1E-06
-    annotation_prokka_coverage      = 80
+    annotation_prodigal_singlemode      = false
+    annotation_prodigal_closed          = false
+    annotation_prodigal_transtable      = 11
+    annotation_prodigal_forcenonsd      = false
+    
+    annotation_bakta_db                 = null
+    annotation_bakta_mincontiglen       = 1
+    annotation_bakta_translationtable   = 11
+    annotation_bakta_gram               = '?'
+    annotation_bakta_complete           = false
+    annotation_bakta_keepcontigheaders  = true
```

Comment:
We should never set a default to `true` as a parameter, otherwise we have to use the less-intuitive `--anotation_bakta_keepcontigheaders=false` way of turning it off.

Flip parameter so while we prefer to keep them, a user can opt-in to rename them: `--annotation_bakta_renamecontigheaders = false`, so if they specify that flag it _will_ rename the headers.